hikari/api/event_manager.py

```diff
@@ -30,27 +30,24 @@
 import typing
 
 from hikari import iterators
+from hikari.events import base_events
 
 if typing.TYPE_CHECKING:
     import types
 
     from hikari.api import shard as gateway_shard
-    from hikari.events import base_events
     from hikari.internal import data_binding
 
-    _T = typing.TypeVar("_T")
-    EventT_co = typing.TypeVar("EventT_co", bound=base_events.Event, covariant=True)
-    EventT_inv = typing.TypeVar("EventT_inv", bound=base_events.Event)
-    PredicateT = typing.Callable[[EventT_co], bool]
-    CallbackT = typing.Callable[[EventT_inv], typing.Coroutine[typing.Any, typing.Any, None]]
+    PredicateT = typing.Callable[[base_events.EventT], bool]
+    CallbackT = typing.Callable[[base_events.EventT], typing.Coroutine[typing.Any, typing.Any, None]]
     ConsumerT = typing.Callable[
         [gateway_shard.GatewayShard, data_binding.JSONObject], typing.Coroutine[typing.Any, typing.Any, None]
     ]
 
-EventT = typing.TypeVar("EventT", bound="base_events.Event")
+    EventStreamT = typing.TypeVar("EventStreamT")
```

Comment:
Switching from a private `_T` type var to a public typevar called `EventStreamT` which isn't actually bound by EventStream seems like an odd change (even if it doesn't have to be bound for its use in this module)

Reply:
Sorry, started playing around yesterday trying to improve typehints more but no luck. This is a left-over from that, as I was using _T for something else. Will change back

Reply:
I don't think the last push actually changed it back

Reply:
Should be done now